typescript/sdk/src/utils/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "@typescript-eslint/explicit-module-boundary-types": ["off"]
```

Comment:
what is this?

Reply:
The SDK package has a lint rule on that we have to explicitly declare types everywhere, which will make for better type artifacts and auto-gen documentation. But utils is still a bit of a mess so I turned it off there for now